ddl/db_change_test.go

```diff
@@ -90,7 +91,8 @@ func (s *testStateChangeSuiteBase) TearDownSuite(c *C) {
 }
 
 // TestShowCreateTable tests the result of "show create table" when we are running "add index" or "add column".
-func (s *testStateChangeSuite) TestShowCreateTable(c *C) {
+func (s *serialTestStateChangeSuite) TestShowCreateTable(c *C) {
+	config.GetGlobalConfig().Experimental.AllowsExpressionIndex = true
```

Comment:
This will affect other test cases. It should be reverted after the case.

Reply:
It's ok not to revert it. Since we always set this flag when using it.

Reply:
If this flag is always enabled, please set it in `init()` function.
This way will cause a problem, when run a single test case, it will pass, but when run all test in DDL package, sometimes anther case may fail, but it cannot know the global flag set in this case will affect others.
Means, The flag enabled in a test case, should not have any side effect to other test cases.